### ISSUE TITLE
GG Max model import tool bug

### ISSUE THREAD

**Issue by Peri32** [TheGameCreators/GameGuruRepo#5596]
Hi,

I'm having trouble with the Max import tool. When I import models and use the rotation tool within the importer, the models seem to lose their collision data. This happens when I place the models in the map and enable physics at the test level - the models fall through the floor as if they have no collision, 

I made few tests and if I don't use the rotation is all normal.

**Comment by Necrym59** [comment]
Verified

**Comment by LeeBamberTGC** [comment]
@Peri32 Can you attach here one of your models that when rotated ruin the collision, thanks!

**Comment by Peri32** [comment]
It happen with every model I tried for example the stock Max Jeep, if you load it into the import tool and rotate it 180 and on the map turn on the phiysics it will just fall into the ground
![1](https://github.com/TheGameCreators/GameGuruRepo/assets/43548667/703e5a39-b2fb-414f-a300-c5134288762a)

**Comment by Kasseyus** [comment]
I’m currently in the process of going through all the outstanding issues.  As this ticket is over a month old, I'm just checking to see if you are still having a problem with this, or if the matter has since been resolved, either through an update to the engine, or if the issue was resolved when another, similar issue was fixed.

Please let me know by replying to this issue and I will investigate further.

**Comment by Peri32** [comment]
Hi thanks for looking into it, I will try to test again the issue with the
last update soon

בתאריך שבת, 12 באוק׳ 2024, 2:16, מאת Kasseyus ***@***.***>:

**Comment by Peri32** [comment]
@Kasseyus  Hi, the bug still exists. When I import a vehicle model as it is, it has collision, but when I rotate it 180 degrees in the import tool, it loses its collision. Although it no longer falls through the ground like before, it still doesn't have collision with the player.

**Comment by Kasseyus** [comment]
@Peri32 Is this a custom model?

**Comment by Peri32** [comment]
@Kasseyus Yes but it happen also if I reimport the stock Max's jeep, when I rotate the model 180 in the import tool it just lose it collision

**Comment by Kasseyus** [comment]
Hello,

  Can you please confirm if you are using the DevTest, Experimental or Standard version of MAX when encountering this issue?

**Comment by Peri32** [comment]
Standard version

‫בתאריך יום א׳, 20 באוק׳ 2024 ב-15:59 מאת ‪Kasseyus‬‏ <‪
***@***.***‬‏>:‬

**Comment by Kasseyus** [comment]
@LeeBamberTGC 
I have successfully reproduced this issue...

The imported jeep (stock asset, MAX collection) does not loose collision, but rather the physics shape does not rotate correctly during the import process..

![image](https://github.com/user-attachments/assets/07693c13-9e0b-4c0a-ac0a-4c61939c2b31)
Imported with 45d rotation

![image](https://github.com/user-attachments/assets/ec4b59f6-1e62-4f0a-b096-e1bdd843bb01)
Imported Jeep Settings

![image](https://github.com/user-attachments/assets/ae0c05e0-1b9d-4b6f-b651-b2eba59f2f20)
Imported with 180d rotation

To note, this does not affect every object the same.  Tests run with imported LowPoly objects, both FBX and DBO sources did not re-create the issue.

**Comment by LeeBamberTGC** [comment]
@Kasseyus Try with the latest DEVTEST with the JEEP DBO model imported, I rotated 45 degrees on the X during the import and in both polygon and convex hull the collision wireframe perfectly matches the model:

![Image](https://github.com/user-attachments/assets/35c2a5d9-0799-492b-8e28-bba4ab6c74e7)

![Image](https://github.com/user-attachments/assets/71d7a509-b290-44e1-b610-1e9d5f439a4c)

**Comment by Peri32** [comment]
I like to provide a video, let me 2-3 days please :)

בתאריך יום ג׳, 22 באפר׳ 2025, 0:20, מאת Lee Bamber ‏<
***@***.***>:
